### PR TITLE
Docs for WAN port connection control to TP-Link Omada gateways

### PR DESCRIPTION
**Pull request by MarkGodwin**
## Proposed change

Documentation for some new entities being added to the TP-Link Omada integration in [#111431](https://github.com/home-assistant/core/pull/111431). All self explanatory entities.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: [#111431](https://github.com/home-assistant/core/pull/111431)
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/tplink_omada.markdown [hunk 1]
```diff
@@ -37,26 +37,29 @@ TP-Link Omada Controller:
...
 
 ### Internet Gateways
 
-- Firmware Update entities.
-- WAN/LAN Port connectivity sensor entities.
+- WAN/LAN Port connectivity sensors
+- WAN Port Online detection sensors
+- WAN Port Connect/Disconnect switches
+- LAN Port PoE activity sensor
+- Firmware Update entities
```
Comment (klaasnicolaas) [inline source/_integrations/tplink_omada.markdown]:
```suggestion
- Firmware Update
```

File: source/_integrations/tplink_omada.markdown [hunk 2]
```diff
@@ -37,26 +37,29 @@ TP-Link Omada Controller:
...
 
 ## Device trackers
 
-- Not currently supported.
+- Not currently supported
```
Comment (klaasnicolaas) [inline source/_integrations/tplink_omada.markdown]: If it's not currently supported, wouldn't it be better to remove it?

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_integrations/tplink_omada.markdown (hunk 1 above)
Comment (MarkGodwin) [reply source/_integrations/tplink_omada.markdown]: Done. Thanks!

File: source/_integrations/tplink_omada.markdown (hunk 2 above)
Comment (MarkGodwin) [reply source/_integrations/tplink_omada.markdown]: Fair point. Removed.

**Review by klaasnicolaas — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged 👍

**Pull request merged**